splunk_add_on_ucc_framework/global_config_validator.py

```diff
@@ -224,10 +224,73 @@ def _validate_validators(self):
             entities = service["entity"]
             for entity in entities:
                 self._validate_entity_validators(entity)
+    @staticmethod
+    def _find_duplicates_in_list(_list) -> list:
+        dup = [x for x in _list if _list.count(x) > 1]
+        return dup if dup else False
+
+
+    def _validate_autoCompleteFields_duplicates(self, options) -> None:
+        labels, values = [], []
+        for field in options["autoCompleteFields"]:
+            labels.append(field["label"])
+            values.append(field["value"])
+        if self._find_duplicates_in_list(values) or self._find_duplicates_in_list(labels):
+            raise GlobalConfigValidatorException(
+                f"`Duplicates found for autoCompleteFields: {options['autoCompleteFields']}"
+            )
+
+    def _validate_entity_duplicates(self, entity) -> None:
+        fields, labels = [], []
+        for _entity in entity:
+            fields.append(_entity['field'])
+            labels.append(_entity['label'])
+            temp = _entity.get("options")
```

Comment:
I think `options` is better name than `temp`.

Reply:
changed to `optio s`